src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/MicrosoftCodeQualityAnalyzersResources.resx

```diff
@@ -1311,6 +1311,14 @@
   <data name="OperatorOverloadsHaveNamedAlternatesCodeFixTitle" xml:space="preserve">
     <value>Add operator overload named alternate</value>
   </data>
+  <data name="AvoidMultipleEnumerationsMessage" xml:space="preserve">
+    <value>Possible multiple enumerations of 'IEnumerable' collection in this method. To prevent unnecessary enumerations, consider caching it first by calling a method such as 'ToList' or 'ToArray'.</value>
```

Comment:
NIT: In the API review there were suggestion that `caching it first by calling a method such as 'ToList' or 'ToArray'` is not always the fix, and `it should have a comprenhensive doc which would suggest different workarounds/fixes`. So i think we should not suggest this fix in the message, maybe replace with something like `see docs for suggestions`?

Reply:
It looks like the analyzer diagnosis already has a link in it
![image](https://user-images.githubusercontent.com/24360909/157553586-6650fa8f-8a00-42fa-9c97-5aa7f8486a79.png)
Then sounds like `Possible multiple enumerations of 'IEnumerable' collection` is enough?

Reply:
Yes it has link, I meant suggesting to see docs for more info, not adding an extra link, that is not necessary, how is this sounds: `Possible multiple enumerations of 'IEnumerable' collection in this method. Consider a design that avoid multiple enumerations`

Reply:
I change this to 
`Possible multiple enumerations of 'IEnumerable' collection in this method. Consider using an implementation that avoid multiple enumerations`